### PR TITLE
VZ-7908: Add more details for ISM policy support by Verrazzano

### PR DESCRIPTION
**Pull request by adiforluls**
PR to add more information about ISM policy created through Verrazzano.

### CODE REVIEW

File: content/en/docs/customize/opensearch.md [hunk 1]
```diff
@@ -115,6 +115,14 @@ Containers:
 Policies can be used to automatically rollover and prune old data, preventing your OpenSearch
 cluster from running out of disk space.
 
+Verrazzano allows you configure OpenSearch index state management (ISM) policies through the Verrazzano custom resource.
```
Comment (anders-swanson) [inline content/en/docs/customize/opensearch.md:118]: We should probably backlink to OpenSearch ISM policies for the opensearch version used in this version of Verrazzano.

File: content/en/docs/customize/opensearch.md [hunk 2]
```diff
@@ -115,6 +115,14 @@ Containers:
...
+The ISM policy created by Verrazzano will contain two states: ingest and delete. The ingest state can only be configured for the rollover action.
+Based on the rollover configuration provided in the Verrazzano custom resource, the rollover action will be configured for the ingest state.
+
+**NOTE:** The ISM policy created via the Verrazzano custom resource contains a minimal set of configurations. To create a more defined ISM policy, the OpenSearch
```
Comment (anders-swanson) [inline content/en/docs/customize/opensearch.md:122]: Can you add an example of how to add more detailed ISM policies. 
Also: more defined ISM policy => more detailed ISM policy

Comment (adiforluls) [reply content/en/docs/customize/opensearch.md]: Done

File: content/en/docs/customize/opensearch.md (hunk 1 above)
Comment (adiforluls) [reply content/en/docs/customize/opensearch.md:118]: We were already doing that, albeit to a different version of OS. Fixed.

File: content/en/docs/customize/opensearch.md [hunk 3]
```diff
@@ -111,10 +111,14 @@ Containers:
 
 ## Configure Index State Management policies
 
-[Index State Management](https://opensearch.org/docs/1.3/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
+[Index State Management](https://opensearch.org/docs/1.2/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
```
Comment (anders-swanson) [inline content/en/docs/customize/opensearch.md:114]: There's an entry in params.toml for the opensearch docs URL

**Review by anders-swanson — COMMENTED** [review]
One minor comment, otherwise looks good. Let me know when you have updated (and resolved merge conflicts), and I'll do the approval.

**Comment by adiforluls** [comment]
> One minor comment, otherwise looks good. Let me know when you have updated (and resolved merge conflicts), and I'll do the approval.

All fixed

**Review by anders-swanson — APPROVED** [review]

File: content/en/docs/customize/opensearch.md [hunk 4]
```diff
@@ -111,7 +111,7 @@ Containers:
 
 ## Configure Index State Management policies
 
-[Index State Management](https://opensearch.org/docs/1.2/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
+[Index State Management]({{<opensearch_docs_url>}}/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
 Policies can be used to automatically rollover and prune old data, preventing your OpenSearch
```
Comment (rosemarymarano) [inline content/en/docs/customize/opensearch.md:115]: -> You can use policies to automatically rollover and prune old data,

File: content/en/docs/customize/opensearch.md [hunk 5]
```diff
@@ -111,7 +111,7 @@ Containers:
 
 ## Configure Index State Management policies
 
-[Index State Management](https://opensearch.org/docs/1.2/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
+[Index State Management]({{<opensearch_docs_url>}}/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
```
Comment (rosemarymarano) [inline content/en/docs/customize/opensearch.md:114]: -> [Index State Management](https://opensearch.org/docs/1.2/im-plugin/ism/index/) (ISM) policies

File: content/en/docs/customize/opensearch.md [hunk 6]
```diff
@@ -111,7 +111,7 @@ Containers:
...
 Policies can be used to automatically rollover and prune old data, preventing your OpenSearch
 cluster from running out of disk space.
```
Comment (rosemarymarano) [inline content/en/docs/customize/opensearch.md:117]: Re-write these sentences:
-> Verrazzano lets you configure OpenSearch ISM policies using the Verrazzano custom resource.
-> The ingest state can be configured only for the rollover action.
-> The rollover action for the ingest state will be configured based on the rollover configuration provided in the Verrazzano custom resource.

On line 147:
The above Verrazzano custom resource... -> The previous Verrazzano custom resource...

On line 206:
The ISM policy created via the... -> The ISM policy created using the
To create a more detailed ISM policy, the OpenSearch REST API can also be used. -> To create a more detailed ISM policy, you can also use the OpenSearch REST API.

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: content/en/docs/customize/opensearch.md [hunk 7]
```diff
@@ -111,10 +111,14 @@ Containers:
 
 ## Configure Index State Management policies
 
-[Index State Management](https://opensearch.org/docs/1.3/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices.
-Policies can be used to automatically rollover and prune old data, preventing your OpenSearch
+[Index State Management]({{<opensearch_docs_url>}}/im-plugin/ism/index/) (ISM) policies configure OpenSearch to manage the data in your indices.
+You can use policies to automatically rollover and prune old data, preventing your OpenSearch
 cluster from running out of disk space.
 
+Verrazzano lets you configure OpenSearch ISM policies through the Verrazzano custom resource.
```
Comment (rosemarymarano) [inline content/en/docs/customize/opensearch.md:118]: -> using the Verrazzano custom resource.

**Review by rosemarymarano — APPROVED** [review]
One more minor fix, otherwise fine.

**Pull request merged**